### ISSUE TITLE
Leads upgrade - Cybertek

### ISSUE THREAD

**Issue by Cybertekww** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1264]
We dont use the leads much for what they are made for and want to be able to use it for quoting custom built computers for customers. The issue I am having is that the lead only lets me add items that are in the repair category and there is a lot of mucking around to try make it work. What we would like is to be able to create a lead for a custom build, list the parts and then send it to the customer as a quote or convert it to a ticket and invoice.

**Comment by TechTraders** [comment]
have you tried using the estimate system? 

https://help.repairdesk.co/knowledgebase/articles/1174753-how-can-i-create-an-estimate

**Comment by Cybertekww** [comment]
Estimates are just invoices that arent registered to receive payment. What we are currently doing is putting all the info in a spreadsheet with the sku, buy price, sell price, profit etc. and then we put all the details of the spreadsheet in to the system as an invoice, create and estimate and send the estimate to the customer. then we create a ticket so we can track everything and leave notes etc. for the job and make sure everyone knows what is happening .

What would be awesome is if in the leads area I could select the lead, create the customer and add in details like the price the customer is wanting to spend (Lead value) th parts they are looking for and go from there. then we dont need to spend a lot of time stuffing around with spreadsheets, estimates, invoices, tickets and what have you

**Comment by TechTraders** [comment]
Fair enough 😄 just thought to check since we just use the estimate system for all of our build quotes and it seems to work alright for us. but I do see the value in what you are suggesting though. I guess we just worked with what we had haha

**Comment by Uzair-RepairDesk** [comment]
@Cybertekww 
We have introduced our new "Estimate" module that covers most of your requirements. 

- A completely new section is created to show estimates’ records and details.
- On the estimate listing page, you can view all the previously created estimates with their relevant status.
- Whenever an estimate is created, the customer will be notified via email.
- Customers can either view, approve, or ask for change requests after reviewing the estimate from the customer portal.
- On approval, it will ask for the customer signatures and the estimate status will be updated.
- On change request, a popup will appear where customers can write their required changes.
- Once the estimate is approved, the repair store’s employees can create a ticket from those estimates.

Please review at your end and let me know if it works for you.
